src/rdb.h

```diff
@@ -42,15 +42,15 @@
  * RDB 11 is the last open-source Redis RDB version, used by Valkey 7.x and 8.x.
  *
  * RDB 12+ are non-open-source Redis formats.
+ * VALKEY 80+ are Valkey open-source versions.
```

Comment:
Mention which Valkey versions use this? We already mentioned above that Valkey 7.x and 8.x use RDB 11.

```suggestion
 * VALKEY 80+ are Valkey >= 9.0.
```

No need to mention open source. All Valkeys are open source. :)

Btw now that Redis added AGPL, maybe we should change the comment about Redis to say restrictive-licensed or non-BSD licenced?